vmdk_plugin/drivers/vmdk/vmdk_driver.go

```diff
@@ -41,8 +41,8 @@ const (
 	devWaitTimeout   = 1 * time.Second
 	sleepBeforeMount = 1 * time.Second
 	watchPath        = "/dev/disk/by-path"
-	version          = "VMDK Volume Driver v0.3"
-	driverName       = "vmdk"
+	version          = "vSphere Volume Driver v0.3"
```

Comment:
I'd suggest 0.11 to catch up with the release #. But if you prefer 0.4 to stay sequential it's good too